docs/driver-only-builds.md

```diff
@@ -240,3 +241,26 @@ removing builtin support (i.e. `MBEDTLS_DHM_C`).
 ### Limitations
 Support for deterministic derivation of a DH keypair
 (i.e. `PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_DERIVE`) is not supported.
+
+AEADs
+-----
+
+It is possible to have all AEADs operations provided only by a driver.
```

Comment:
I don't disagree, but that's G3 and this is supposed to be G2 wrap-up :)

Maybe skip this commit for now and work on the text in the G3-G4 wrap-up issue instead? The situation is actually not that easy to describe in a way that's both complete and readable.